src/shared-components/dropdown/style.js

```diff
@@ -101,10 +99,10 @@ export const DropdownOptionsContainer = styled.ul`
   list-style: none;
   -webkit-overflow-scrolling: touch;
 
-  ${({ isOpen }) =>
+  ${({ isOpen, maxHeight }) =>
```

Comment:
It's verbose, but if you're not opposed I think we might prefer `optionsContainerMaxHeight` instead of `maxHeight`. It's wordy but it's also clearer that it applies to the options container specifically, and makes discoverability a bit easier.

Reply:
I like that idea. I'm always in favor of verbosity if it increases clarity 👍